internal/app/machined/pkg/controllers/kubespan/nftables.go

```diff
@@ -258,17 +297,24 @@ func matchIPv6Set(set *nftables.Set, mark, mask uint32) []expr.Any {
 	return matchIPSet(set, mark, mask, nftables.TableFamilyIPv6)
 }
 
-func matchIPSet(set *nftables.Set, mark, mask uint32, family nftables.TableFamily) []expr.Any {
-	var (
-		offset uint32 = 16
-		length uint32 = 4
-	)
-
-	if family == nftables.TableFamilyIPv6 {
+func ipOffsetLength(family nftables.TableFamily) (offset uint32, length uint32) {
+	switch family { //nolint:exhaustive
+	case nftables.TableFamilyIPv4:
+		offset = 16
+		length = 4
+	case nftables.TableFamilyIPv6:
 		offset = 24
 		length = 16
+	default:
+		panic("unexpected IP family")
```

Comment:
should we really panic here?

Reply:
yes, we call this code (no user input), and it's tested, so if it panics, something went wrong